src/Google.Api.Gax.Grpc/BidirectionalStreamingBase.cs

```diff
@@ -88,9 +88,9 @@ public virtual Task WriteAsync(TRequest message, WriteOptions options)
         /// Completes the stream when all buffered messages have been sent. This method can only be called
         /// once, and further messages cannot be written after it has been called.
```

Comment:
TryWriteComplete can be called multiple times, but only the first time will have any effect (at the moment - that will change when we put it in the buffer, of course).